test/test-parse-typescript-class.cpp

```diff
@@ -1933,6 +1933,80 @@ TEST_F(test_parse_typescript_class,
         }));
   }
 }
+
+TEST_F(test_parse_typescript_class,
+       class_keyword_with_escape_sequence) {
```

Comment:
Nit: I think this test is misnamed:
```suggestion
       constructor_keyword_with_escape_sequence) {
```

Reply:
Resolved.